src/main/java/uk/gov/hmcts/cmc/claimstore/events/claim/DocumentGenerator.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.cmc.claimstore.events.claim;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.documents.CitizenSealedClaimPdfService;
+import uk.gov.hmcts.cmc.claimstore.documents.DefendantPinLetterPdfService;
+import uk.gov.hmcts.cmc.claimstore.documents.LegalSealedClaimPdfService;
+import uk.gov.hmcts.cmc.claimstore.documents.output.PDF;
+import uk.gov.hmcts.cmc.claimstore.events.DocumentGeneratedEvent;
+import uk.gov.hmcts.cmc.claimstore.events.solicitor.RepresentedClaimIssuedEvent;
+import uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils;
+
+import static java.lang.String.format;
+
+@Component
+public class DocumentGenerator {
+
+    private final CitizenSealedClaimPdfService citizenSealedClaimPdfService;
+    private final DefendantPinLetterPdfService defendantPinLetterPdfService;
+    private final LegalSealedClaimPdfService legalSealedClaimPdfService;
+    private final ApplicationEventPublisher publisher;
+
+    @Autowired
+    public DocumentGenerator(final CitizenSealedClaimPdfService citizenSealedClaimPdfService,
```

Comment:
could you move the parameter to a new line please?
same with last parameter moving the `) {`  down

Reply:
Done 👍

Reply:
Done 👍